api/index.js

```diff
@@ -1,49 +1,34 @@
-require('dotenv').config()
 const { SESSION_NAME, SESSION_MAX_AGE, SECRET, SITE_URL } = process.env
 
-const { send } = require('micro')
-const morgan = require('micro-morgan')
-const redirect = require('micro-redirect')
 const session = require('micro-cookie-session')({
     name: SESSION_NAME,
     keys: [SECRET],
     maxAge: SESSION_MAX_AGE * 60 * 1000,
 })
 
-const md5 = require('md5')
 const emptygif = require('emptygif')
 const encodeUrl = require('encodeurl')
 
 const track = require('./lib/track')
 const queryParser = require('./lib/queryParser')
 
-module.exports = morgan('tiny')(async (req, res) => {
-    if (req.url === '/robots.txt') {
-        console.log('Sending robots response')
-        return send(res, 200, ['User-agent: *', 'Disallow: /'].join("\n"))
-    }
-    if (req.url === '/favicon.ico') {
```

Comment:
Removed handling of this request as it should really be handled by the `rewrite` rules in the `now.json` configuration, just like the `robots.txt` file. Felt like it was a bit redundant though so left it out